index.js

```diff
@@ -78,7 +78,14 @@ else if (program.translator && program.hub) {
                 var dInfo = { 'deviceInfo': deviceInfo, 'hub': hub };
                 
                 if (program.get) {
-                    translatorCli.getProperty(program.translator, dInfo, program.get, true).then(info => {
+                    
+                    // If a device/entity id was provided, then pass it, otherwise pass expand=true
+                    var value = true;
```

Comment:
di is only passed on getResource, and expand is only passed on get and getPlatform so they are mutually exclusive.  The cli always sets expand to true for getplatform.